README.md

```diff
@@ -48,11 +54,11 @@ Set to `true` to start in Insert mode when opening files.
 
 Set to `true` to will make line numbers relative when not in Insert mode.
 
-#### `amVim.mimicVimSearchBehavior`
+#### `amVim.useSystemClipboard`
 
-`Boolean`, Default: `true`
+`Boolean`, Default: `false`
 
-Set to `false` to keep VSCode's keybinding when searching.
+Set to `true` to copy to and paste from the system clipboard.
```

Comment:
Looking at it, now i wonder if it might be more "vim-ish" to say "yank to" instead of "copy to". Documentation is hard 🤣